library/src/main/kotlin/it/krzeminski/githubactions/yaml/JobsToYaml.kt

```diff
@@ -56,6 +56,11 @@ private fun Job.toYaml() = buildString {
         }
     }
 
+    job.timeoutMinutes?.let { value ->
+        require(value > 0) { """Invalid value of field Job(timeoutMinutes=$value)""" }
```

Comment:
Let's be more specific here:
```suggestion
        require(value > 0) { """timeout should be positive""" }
```
and I'd also move this validation out of YAML serialization logic. It should happen as soon as the job object is created.

Reply:
i followed the same pattern as the name validation

Reply:
Ok, so we should change the name validation :) I can do it in spare time.

Reply:
#144